Source/UI/VirtusizeButton.swift

```diff
@@ -26,7 +26,7 @@
 public class VirtusizeButton: UIButton, VirtusizeView, VirtusizeViewEventProtocol {
 	public var presentingViewController: UIViewController?
 	public var messageHandler: VirtusizeMessageHandler?
-	public var product: VirtusizeProduct?
+	public var clientProduct: VirtusizeProduct?
```

Comment:
@akueisara  💅🏻 just a small suggestion but we use `user` as in `userProduct` instead of `client` in aoyama

Reply:
but I see it in many places so I don't want to cause you an extra work

Reply:
@Ania-chan Thank you for your suggestion. I use clientProduct for Flutter and Android SDK as well, so I will keep it as clientProduct for now. 
I name it as a clientProduct because it caches the data like an external product ID and a product image URL mainly from our client, as opposed to a serverProduct, which is meant to cache the data fetched from our server.